tgfilestream/telegram.py

```diff
@@ -16,20 +16,22 @@
 import logging
 
 from telethon import TelegramClient, events
+from telethon.sessions import StringSession
 
 from .paralleltransfer import ParallelTransferrer
-from .config import session_name, api_id, api_hash, public_url
+from .config import session_name, api_id, api_hash, public_url, start_message
 from .util import pack_id, get_file_name
 
 log = logging.getLogger(__name__)
 
-client = TelegramClient(session_name, api_id, api_hash)
+client = TelegramClient(StringSession(session_name), api_id, api_hash)
```

Comment:
This should be an option rather than the default, maybe a `TG_STRING_SESSION` env var that's used instead of session_name if set?

Something like this (still need to define `string_session` of course)
```suggestion
if string_session:
    session = StringSession(string_session)
else:
    session = session_name
client = TelegramClient(session, api_id, api_hash)
```